dwave/system/composites/linear_ancilla.py

```diff
@@ -0,0 +1,214 @@
+                )  # TODO: implement larger biases through multiple ancillas
+
+            available_ancillas = set(g_target.adj[variable]) - set(g_source.nodes())
+            if not len(available_ancillas):
+                raise ValueError(f"variable {variable} has no ancillas available")
+            unused_ancillas = available_ancillas - set(used_ancillas)
+            if len(unused_ancillas):
+                ancilla = unused_ancillas.pop()
+                # bias sign is handled by the flux bias
+                flux_biases[ancilla] = np.sign(bias) * largest_j_sign * fb_magnitude
+                _bqm.add_interaction(
+                    variable, ancilla, (abs(bias) - h_tolerance) * largest_j_sign
+                )
+            else:
+                if qpu_properties["j_range"][0] <= bias <= qpu_properties["j_range"][1]:
+                    # If j can be sign-flipped, select the least used ancilla
+                    ancilla = sorted(
+                        list(available_ancillas), key=lambda x: len(used_ancillas[x])
+                    )[0]
+                    _bqm.add_interaction(
+                        variable,
+                        ancilla,
+                        (bias - h_tolerance * np.sign(bias))
+                        * np.sign([flux_biases[ancilla]]),
+                    )
+                else:
+                    # Ancilla sharing is limited to flux biases with a  sign
+                    signed_ancillas = [
+                        ancilla
+                        for ancilla in available_ancillas
+                        if largest_j_sign
+                        == np.sign(flux_biases[ancilla]) * np.sign(bias)
+                    ]
+                    if not len(signed_ancillas):
+                        return ValueError(
+                            f"variable {variable} has no ancillas available"
+                        )
+                    else:
+                        ancilla = sorted(
+                            list(signed_ancillas), key=lambda x: len(used_ancillas[x])
+                        )[0]
+                        _bqm.add_interaction(
+                            variable,
+                            ancilla,
+                            largest_j_sign * (abs(bias) - h_tolerance),
+                        )
+
+            used_ancillas[ancilla].append(variable)
+            _bqm.set_linear(variable, h_tolerance * np.sign(bias))
+
+        sampleset = self.child.sample(_bqm, flux_biases=flux_biases, **parameters)
+        yield
+        yield dimod.SampleSet.from_samples_bqm(
+            [
+                {k: v for k, v in sample.items() if k not in used_ancillas}
+                for sample in sampleset.samples()
+            ],
+            bqm=bqm,
+            info=sampleset.info.update(used_ancillas),
+        )
+
+
+def innermost_child_properties(sampler: dimod.Sampler) -> Mapping[str, Any]:
```

Comment:
Let's mark this as private
```suggestion
def _innermost_child_properties(sampler: dimod.Sampler) -> Mapping[str, Any]:
```